old/output.json

```diff
@@ -1,121 +1,121 @@
+                    "absolutes": 24.480056409646838
+                }, 
+                "pubpol": {
+                    "cohorts": [], 
+                    "current": {
+                        "percentage": 61.111111111111114, 
+                        "name": "2014", 
+                        "value": 4.875
+                    }, 
+                    "segmentations": [], 
+                    "past_results": [], 
+                    "type": "percentileChart", 
+                    "absolutes": 22.541384351683806
+                }, 
+                "comimp": {
+                    "cohorts": [], 
+                    "current": {
+                        "percentage": 38.888888888888886, 
+                        "name": "2014", 
+                        "value": 5.7445652173913047
+                    }, 
+                    "segmentations": [], 
+                    "past_results": [], 
+                    "type": "percentileChart", 
+                    "absolutes": 29.008593737868345
+                }, 
+                "undrwr": {
+                    "cohorts": [], 
+                    "current": {
+                        "percentage": 27.777777777777771, 
+                        "name": "2014", 
+                        "value": 5.6956521739130439
+                    }, 
+                    "segmentations": [], 
+                    "past_results": [], 
+                    "type": "percentileChart", 
+                    "absolutes": 29.228986431748382
+                }, 
+                "undrsoc": {
+                    "cohorts": [], 
+                    "current": {
+                        "percentage": 61.111111111111114, 
+                        "name": "2014", 
+                        "value": 5.8894009216589858
+                    }, 
+                    "segmentations": [], 
+                    "past_results": [], 
+                    "type": "percentileChart", 
+                    "absolutes": 28.556907829013092
+                }, 
+                "orgimp": {
+                    "cohorts": [], 
+                    "current": {
+                        "percentage": 83.333333333333329, 
+                        "name": "2014", 
+                        "value": 6.253393665158371
+                    }, 
+                    "segmentations": [], 
+                    "past_results": [], 
+                    "type": "percentileChart", 
+                    "absolutes": 30.256967284521462
+                }, 
+                "impsust": {
+                    "cohorts": [], 
+                    "current": {
+                        "percentage": 50.0, 
+                        "name": "2014", 
+                        "value": 5.3467336683417086
+                    }, 
+                    "segmentations": [], 
+                    "past_results": [], 
+                    "type": "percentileChart", 
+                    "absolutes": 26.663064818798496

```

Comment:
`absolutes` should be a list of 5 values. Something is incorrect in your exercise 3. This is where unit tests would come in handy.